src/api/roles/coach/coach.py

```
@@ -675,6 +677,122 @@ def get_coach_earnings(
 
     return CoachEarningsResponse(total_earnings=total, since=since)
 
+@router.get("/my_clients")
+def get_my_clients(db = Depends(get_session), acc: Account = Depends(get_coach_account)):
```

Ruff (0.15.12)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)